app/views/spree/shared/_review_summary.html.erb

```diff
@@ -1,9 +1,8 @@
 <div id="reviews">
-  <%= render 'spree/shared/rating', locals: @product, review: 0 %>
-  <% for review in (Spree::Reviews::Config[:track_locale] ? @product.reviews.localized(I18n.locale) : @product.reviews).default_approval_filter.preview %>
-    <%= render 'spree/shared/review', review: review %>
-  <% end %>
-  <% if Spree::Reviews::Config[:feedback_rating] && (!Spree::Reviews::Config[:require_login] || spree_current_user) %>
```

Comment:
You forgot to add this condition inside the check "!Spree::Reviews::Config[:require_login]"